txt2html.py

```diff
@@ -205,6 +206,18 @@ def main():
     if args.lang:
         lang_attribute_value = args.lang
 
+    if args.config:
+        config_file_path = args.config
+        try:
+            with open(config_file_path, "rb") as config_file:
+                config_data = tomli.load(config_file)
+
+                lang_attribute_value = config_data.get("lang", lang_attribute_value)
+        except FileNotFoundError:
+            print(f"Config file not found: {config_file_path}")
```

Comment:
```suggestion
            parser.error(f"Config file not found: {config_file_path}")
```

We need to exit here if If the file is missing, or can't be parsed as TOML

Reply:
sure